public/style/components/_animation.scss

```diff
@@ -0,0 +1,115 @@
+svg .snow {
+  animation: snow-fall 5s linear infinite;
+  fill: $blue-light;
+  transform-origin: center;
+}
+
+@keyframes snow-fall {
+
+  0% {
+    opacity: 0;
+    transform: translate3d(0, -20%, 0) rotateZ(0deg);
+  }
+
+  25% {
+    opacity: .5;
+    transform: translate3d(0, -15%, 0) rotateZ(15deg);
+  }
+
+  50% {
+    opacity: 1;
+    transform: translate3d(0, -10%, 0) rotateZ(30deg);
+  }
+
+  75% {
+    opacity: 1;
+    transform: translate3d(0, 0%, 0) rotateZ(45deg);
+  }
+
+  100% {
+    opacity: 0;
+    transform: translate3d(0, 10%, 0) rotateZ(60deg);
+  }
+
+}
+
+svg .rain,
+svg .hail {
+  animation: rain 5s ease-in infinite;
+
+  &:nth-child(2) {
+    animation-delay: 0s;
+  }
+
+  &:nth-child(3) {
+    animation-delay: .5s;
+  }
+
+  &:nth-child(4) {
+    animation-delay: .75s;
+  }
+}
+
+svg .bolt {
+  animation: bolt-flash 2s ease-in infinite;
+}
+
+@keyframes rain {
+  0% {
+    opacity: 0;
+    transform: translateY(-.1em);
+  }
+
+  70% {
+    opacity: 1;
+  }
+
+  90% {
+    opacity: .9;
+  }
+
+  100% {
+    opacity: 0;
+    transform: translateY(0);
+  }
+}
+
+@keyframes bolt-flash {
+  0% {
+    opacity: 1;
+    transform: translateY(-.2em);
+  }
+
+  20% {
+    opacity: 0;
+  }
+
+  40% {
+    opacity: 1;
+    transform: translateY(0);
+  }
+
+  100% {
+    opacity: 0;
+  }
+}
+
+// .fade-in {
+//     animation: fade-in .5s ease-in-out;
+// }
+
+@keyframes fade-in {
```

Comment:
These keyframes need to be commented out, it's still being included in the complication even though the class is commented out.